src/PX.Analyzers/PX.Analyzers.Vsix/Utils/UIAttributes/AcuDisplayNameAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.ComponentModel;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using PX.Analyzers.Vsix;
+using PX.Analyzers.Vsix.Utilities;
+
+
+namespace PX.Analyzers.Vsix
+{
+	[AttributeUsage(AttributeTargets.Class | AttributeTargets.Property | AttributeTargets.Field, Inherited = false, AllowMultiple = false)]
+	internal sealed class AcuDisplayNameAttribute : DisplayNameAttribute
```

Comment:
Name doesn't reflect the difference between `DisplayNameAttribute` and this class.